docs/GettingStarted.md

```diff
@@ -816,6 +816,7 @@ Where `options` is an optional `Hash` that accepts the following parameters:
 | `analytics_enabled` | Enable analytics for spans produced by this integration. `true` for on, `nil` to defer to global setting, `false` for off. | `nil` |
 | `enabled` | Defines whether Grape should be traced. Useful for temporarily disabling tracing. `true` or `false` | `true` |
 | `service_name` | Service name used for `grape` instrumentation | `'grape'` |
+| `error_statuses`| Defines a status code or range of status codes which should be marked as errors. `'500-599'` or `['500-599']` | `500-599` |
```

Comment:
small nit, could we also include an example here that shows support multiple ranges, for clarity? It's unclear currently that there can be multiple ranges